pkgs/development/python-modules/autobahn/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildPythonPackage, fetchurl, isPy3k,
+  unittest2, mock, pytest, trollius, pytest-asyncio,
+  six, twisted, txaio
+}:
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  pname = "autobahn";
+  version = "0.18.2";
+
+  src = fetchurl {
+    url = "mirror://pypi/a/${pname}/${name}.tar.gz";
+    sha256 = "1alp71plqnrak5nm2vn9mmkxayjb081c1kihqwf60wdpvv0w7y14";
+  };
+
+  buildInputs = [ unittest2 mock pytest trollius pytest-asyncio ];
+  propagatedBuildInputs = [ six twisted txaio ];
+
+  disabled = !isPy3k;
```

Comment:
Why are you disallowing python2.7 here? The upstream package claims Python2.7 support. This breaks any python package that depends on 2.7 and autobahn.